docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/besluittype_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+### Resource: INFORMATIEOBJECTTYPE
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil informatieobjecttypeinformatie van de informatieobjecten die voor kunnen komen bij het zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het besluittype is opgenomen. |
+| - gevolg       | Een overzicht van Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om details van informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het informatieobjecttype is opgenomen. Tevens moet de identficatie van het informatieobjecttype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| Gegevens       | INFORMATIEOBJECTTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/informatieobjecttype_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+
+### Resource: eigenschap
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil alle eigenschappen van een zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.             |
```

Comment:
Lijkt in tegenspraak met wat bij de tweede raadpleegoperatie staat.

Reply:
Kopieer foutje

Reply:
Is aangepast

Reply:
```suggestion
| Aanleiding     | Een consumer wil een eigenschap of een overzicht van alle eigenschappen opvragen van een zaaktype zoals opgenomen in de catalogus van de organisatie.             |
```

Reply:
Dat voorstel maakt het verhaal onduidelijk en daarmee slechter van kwaliteit. Ik zeg niet doen.

Reply:
@michielverhoef, welke mogelijkheden biedt de API qua Eigenschappen? Alleen maar opvragen van alle eigenschappen van een zaaktype? Dus niet een enkele (één) eigenschap van een zaaktype opvragen? In jouw beschrijving staat alleen het eerste. Dan zou het tweede dus niet ondersteuntd worden door de API. Is dat juist?

Reply:
Ik verwacht dat ook een deel van de eigenschappen van een objectype opgevraagd kan worden. Maar daar kunnen Sergei en/of Joeri beter antwoord op geven.

De aanleiding staat momenteel als volgt omschreven:
"Een consumer wil zaaktypeinformatie opvragen uit de catalogus."

Volgens mij dekt dat de lading voor zowel een lijst van zaaktypen als zaakdetails (alle details of een beperkt aantal).

Reply:
> De aanleiding staat momenteel als volgt omschreven: "Een consumer wil zaaktypeinformatie opvragen uit de catalogus." Volgens mij dekt dat de lading voor zowel een lijst van zaaktypen als zaakdetails (alle details of een beperkt aantal).

Het gaat hier over het objecttype EIGENSCHAP met een n-1-relatie naar ZAAKTYPE. Dus niet over zaaktypen noch over zaakdetails. Een 'list' geeft m.i. een overzicht van alle eigenschappen bij een zaaktype. Een 'read' geeft m.i. alle informatie over één eigenschap bij een zaaktype. 
Je aanleidingtekst klopt dan niet.

Reply:
Op dit moment kan je niet filteren/limiteren op 1 eigenschap, dus je krijgt of de hele lijst van de eigenschappen, of je vraagt de details op voor 1 enkele eigenschap.